src/components/Responsive/Element/Element.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from 'react';
+import Context from '../Context';
+
+const Element: React.FC = ({ children, ...passedProps }) => (
```

Comment:
Did it complain when typing children with something like `children?: React.ReactNode` ?

Reply:
No, `React.FC` handling of the `children` prop is so simple and nice. No complaints at all.